shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/NearbyRepository.kt

```diff
@@ -1,17 +1,34 @@
 package com.mbta.tid.mbta_app.repositories
 
+import com.mbta.tid.mbta_app.model.RoutePattern
 import com.mbta.tid.mbta_app.model.RouteType
+import com.mbta.tid.mbta_app.model.Stop
 import com.mbta.tid.mbta_app.model.response.GlobalResponse
 import com.mbta.tid.mbta_app.model.response.NearbyResponse
 import io.github.dellisd.spatialk.geojson.Position
 import org.koin.core.component.KoinComponent
 
 interface INearbyRepository {
-    fun getStopIdsNearby(global: GlobalResponse, location: Position): List<String>
+    /**
+     * Gets the list of stops within 0.5 miles (or 1 mile for CR). Includes only stops with
+     * [LocationType.STOP].
+     *
+     * @param excludeRedundantService when true, this will omit stops that serves route patterns
+     *   that are all served by closer stops.
+     */
+    fun getStopIdsNearby(
+        global: GlobalResponse,
+        location: Position,
+        excludeRedundantService: Boolean = false
```

Comment:
I think giving this parameter no default value would make it clearer when we still want the old behavior.

Reply:
Yeah that's a good call. Now that the cleanup of old codepaths is done there might not actually be places we still want the old behavior ... will do a check and remove this parameter entirely if it is possible.

Reply:
Removed!